### DISCUSSION TITLE
Working with GSC that has Domain Property

### DISCUSSION THREAD

**Discussion by TedTansley** [joshcarty/google-searchconsole#67] [Q&A] [answered]
Been working with this for a few weeks and it works great on most connected google search console accounts. Though, I run into errors with the accounts that are connected by "Domain Property".

Getting the following response:

> AttributeError: 'NoneType' object has no attribute 'query'

In the past this meant that I got the URL incorrect when passing it into webproperty.

Any thoughts as to why a domain property set up would be any different or what I could do to get around this?

**Comment by joshcarty** [comment, marked as answer]
You can see all of the webproperties in your account with:

```python
>>> account.webproperties
[<searchconsole.account.WebProperty(url='...')>]
```

I think domain properties are prefixed with `sc-domain` but I'd recommend doing the above to see exactly.

**Reply by TedTansley** [reply]
yep, it worked. Thank you!!
